src/routes/tools/_common/helpers.js

```diff
@@ -26,11 +26,11 @@ export function buildEnvelope(_data) {
     (d) => d.date.getUTCFullYear()
   );
   return dataArr.map(([key, value]) => {
-    const sortedArr = sort(value);
+    const arrExtent = extent(value);
```

Comment:
I made this change to create an envelope from more than 2 series. I did not end up using it, but left it in, because I thought this was a better implementation than the previous one.